app/models/concerns/rad_company.rb

```diff
@@ -72,6 +79,12 @@ def usage_stats
     [usage_headers, usage_items, usage_data]
   end
 
+  def sanitize_twilio_numbers
+    return unless RadicalTwilio.twilio_enabled? && Company.has_attribute?(:twilio_phone_numbers)
```

Comment:
all logic involving twilio_phone_numbers would be simplified, any twilio app would have the required env var